src/test/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicatorControlTest.java

```diff
@@ -0,0 +1,589 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aver.ptz;
+
+import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZConstants.HASH;
+import static org.mockito.Mockito.times;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.avispl.symphony.api.dal.dto.control.ControllableProperty;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.Command;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.AEMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.BGainControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.FocusControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PanTiltDrive;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.RGainControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.WBMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.ZoomControl;
+
+/**
+ * Unit test for Control AverPTZ Communicator
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class AverPTZCommunicatorControlTest {
```

Comment:
Can you add some negative cases ?

Reply:
![image](https://user-images.githubusercontent.com/90905186/141765274-b3fec6e9-5983-4430-8870-1ba611562825.png)
I see many tests are failed. Please check!